lib/utilities/monitoringHandler.js

```diff
@@ -44,6 +44,10 @@ const httpRequestDurationSeconds = new client.Histogram({
     labelNames,
     buckets: [0.0001, 0.005, 0.015, 0.05, 0.1, 0.2, 0.3, 0.4, 0.5], // buckets for response time from 0.1ms to 500ms
 });
+const httpActiveRequests = new client.Gauge({
+    name: 'http_active_requests',
+    help: 'Total number of HTTP requests',
```

Comment:
This is miss-leading, it should be something that explicitly describe the value is a gauge and only contains currently active requests.

Reply:
Bad copy-paste indeed, good catch! 👍